src/back-end/lib/db/organization.ts

```diff
@@ -501,3 +533,46 @@ export const readOneOrganizationOwner = tryDb<[Id], User | null>(
     return valid(result ? await rawUserToUser(connection, result) : null);
   }
 );
+
+export const qualifyOrganizationServiceAreas = tryDb<
+  [Id, number[], Session],
```

Comment:
Changing the type of the PK for serviceAreas ended up being a bit of a time sink because quite a few tables relied on it. I have an idea of what I want to do, but might be better to put it off until after we have more work done; though I want to discuss a migration strategy for it when we get to that point.

I did end up using a more informative ServiceAreaId type which aliases number in the meantime.